Common/ShellFileItem.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Files.Common
+{
+    public class ShellFileItem
+    {
+        public bool isFolder;
```

Comment:
UpperCamelCase for public fields

Reply:
Ok fixed the naming convention of a few variables. I'll be more careful 😅